frontend/components/product-list/sections/FilterableProductsSection/useFacets.tsx

```diff
@@ -33,15 +35,65 @@ export function useFilteredFacets(productList: ProductList) {
       );
    }, [productList.wikiInfo]);
 
+   const sortBy = React.useMemo(() => {
+      return sortFacets(productList.type);
+   }, [productList.type]);
+
    const usefulFacets = React.useMemo(() => {
-      const usefulFacets = facets.slice().filter((facet) => {
-         if (facet === 'facet_tags.Item Type' && isItemTypeProductList) {
-            return false;
-         }
-         return !infoNames.has(facet);
-      });
+      const usefulFacets = facets
+         .slice()
+         .filter((facet) => {
+            if (facet === 'facet_tags.Item Type' && isItemTypeProductList) {
+               return false;
+            }
+            return !infoNames.has(facet);
+         })
+         .sort(sortBy);
       return usefulFacets;
-   }, [facets, infoNames, isItemTypeProductList]);
+   }, [facets, infoNames, isItemTypeProductList, sortBy]);
 
    return usefulFacets;
 }
+
+const partsFacetRanking = new Map([
+   ['facet_tags.Item Type', 1],
+   ['facet_tags.Part or Kit', 2],
+]);
+
+const toolsFacetRanking = new Map([
+   ['facet_tags.Tool Category', 1],
+   ['price_range', 2],
+]);
+
+function sortFacets(productListType: ProductListType) {
+   switch (productListType) {
+      case ProductListType.AllParts:
+      case ProductListType.DeviceParts:
+      case ProductListType.DeviceItemTypeParts:
+         return sortFacetsWithRanking(partsFacetRanking);
+      case ProductListType.AllTools:
+      case ProductListType.ToolsCategory:
+         return sortFacetsWithRanking(toolsFacetRanking);
+      default:
+         return sortFacetsAlphabetically;
+   }
+}
+
+function sortFacetsWithRanking(ranking: Map<string, number>) {
+   return (a: string, b: string): number => {
+      if (ranking.get(a) && ranking.get(b)) {
+         return ranking.get(a)! - ranking.get(b)!;
+      } else if (ranking.get(a)) {
+         return -1;
+      } else if (ranking.get(b)) {
+         return 1;
+      } else {
+         return sortFacetsAlphabetically(a, b);
+      }
+   };
```

Comment:
This is a little simpler and allows negative rankings (where a facet could be pinned to the bottom). Plus if two facets have the same rank, they'll be sorted alphabetically. I didn't check this, so the aRank-bRank might be backwards.

```suggestion
   return (aName: string, bName: string): number => {
      const aRank = ranking.get(aName) || 0;
      const bBrank = ranking.get(bName) || 0;
      
      return aRank - bRank || sortFacetsAlphabetically(aName, bName);
   };
```

Reply:
Also, requires changing the mapping values above because higher number = closer to top.